data/en/1_0005_updated.json

```diff
@@ -0,0 +1,2172 @@
+                                        "any redundancy pay and pay in lieu of notice payments (taxable and non-taxable)",
+                                        "trainees on government schemes",
+                                        "director’s fees",
+                                        "employer’s NI and contribution to pension schemes",
+                                        "employees working abroad unless paid directly from this business’s GB payroll",
+                                        "signing on fees (sporting professionals)",
+                                        "accrued holiday pay",
+                                        "employees in Northern Ireland",
+                                        "expenses payments for attending meetings, for example, councillors"
+                                    ]
+                                }
+                            ]
+                        }
+                    }],
+                    "title": "Calendar Monthly Pay",
+                    "type": "Question"
+                },
+                {
+                    "id": "calendar-monthly-pay-breakdown",
+                    "questions": [{
+                        "answers": [{
+                                "id": "calendar-monthly-pay-breakdown-arrears-answer",
+                                "label": "Arrears of pay owing to pay awards",
+                                "mandatory": true,
+                                "q_code": "171",
+                                "type": "Currency",
+                                "currency": "GBP",
+                                "decimal_places": 2,
+                                "max_value": {
+                                    "answer_id": "calendar-monthly-pay-gross-pay-answer",
+                                    "exclusive": true
+                                }
+                            },
+                            {
+                                "id": "calendar-monthly-pay-breakdown-prp-answer",
+                                "label": "Bonuses, commissions or annual profit from profit related pay schemes",
+                                "mandatory": true,
+                                "q_code": "181",
+                                "type": "Currency",
+                                "currency": "GBP",
+                                "decimal_places": 2,
+                                "max_value": {
+                                    "answer_id": "calendar-monthly-pay-gross-pay-answer",
+                                    "exclusive": true
+                                }
+                            }
+                        ],
+                        "id": "calendar-monthly-pay-breakdown-question",
+                        "title": "Of the <em>{{answers.five_weekly_total_gross|format_currency}}</em> paid to <em>five weekly</em> paid employees in {{exercise.period_str}}, what other amounts were paid:",
```

Comment:
I think this is wrong. you are piping `answers.five_weekly_total_gross` into a calendar monthly question?

Reply:
Lazy copying and pasting that is 🙃